fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java

```diff
@@ -77,29 +92,114 @@ public void afterPropertiesSet() throws Exception {
     Assert.state(gitHubProperties.getOrganization() != null, "Supply an organization");
     Assert.state(gitHubProperties.getBaseUrl() != null, "Supply a base url");
 
+    this.initializeMembersCache();
+    this.initializeTeamsCache();
     this.initializeTeamMembershipCache();
   }
 
-  private void initializeTeamMembershipCache() {
-    this.teamMembershipCache = CacheBuilder.newBuilder()
-        .maximumSize(1000)
-        .expireAfterWrite(this.gitHubProperties.getMembershipCacheTTLSeconds(), TimeUnit.SECONDS)
+  private void initializeMembersCache() {
+    this.membersCache = CacheBuilder.newBuilder()
+        .maximumSize(1)
```

Comment:
Uh... max size of 1? If this intentional, I'd add a comment as to why.

Reply:
This is a single item cache of all members in the github org.   Will add a comment.